File: freak-fights-fan/shared/features/teams/responses/TeamDto.ts
import { FighterInTeamDto } from "../../fighters/responses/FighterInTeamDto";

export interface TeamDto {
    id: number;
    created: Date;
    modified: Date;

    Number: number;
    FightId: number;
    FighterInTeams: Array<FighterInTeamDto>;
}

File: FreakFightsFan.Shared/Features/Teams/Responses/TeamDto.cs
using FreakFightsFan.Shared.Features.Fighters.Responses;

namespace FreakFightsFan.Shared.Features.Teams.Responses;

public class TeamDto
{
    public int Id { get; set; }
    public DateTime Created { get; set; }
    public DateTime Modified { get; set; }

    public int Number { get; set; }
    public int FightId { get; set; }
    public List<FighterInTeamDto> FighterInTeams { get; set; }
}